tests/Users/create-user.spec.js

```diff
@@ -0,0 +1,129 @@
+import { test, expect } from "@playwright/test";
+
+const getRandomNumber = () => {
+  return Math.floor(Math.random() * (999 - 100 + 1) + 100);
+};
+
+const wait = (seconds) => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, seconds * 1000);
+  });
+};
+
+test.describe("Create user tests", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("./");
+    await page.getByPlaceholder("Email Address").fill("root@gmail.com");
+    await page.getByPlaceholder("Password").fill("p@s5W07d");
+    await page.getByRole("button", { name: "Log in" }).click();
+  });
```

Comment:
The login steps in beforeEach and the steps to navigate to the "Create user" page are repeated across tests. Consider extracting these steps into reusable functions to reduce code duplication and improve test maintainability.

Reply:
fixed